include/boost/geometry/algorithms/detail/overlay/get_distance_measure.hpp

```diff
@@ -0,0 +1,188 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2019 Barend Gehrels, Amsterdam, the Netherlands.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_ALGORITHMS_DETAIL_OVERLAY_GET_DISTANCE_MEASURE_HPP
+#define BOOST_GEOMETRY_ALGORITHMS_DETAIL_OVERLAY_GET_DISTANCE_MEASURE_HPP
+
+#include <boost/geometry/core/access.hpp>
+#include <boost/geometry/core/coordinate_system.hpp>
+#include <boost/geometry/core/coordinate_type.hpp>
+#include <boost/geometry/util/select_coordinate_type.hpp>
+
+namespace boost { namespace geometry
+{
+
+#ifndef DOXYGEN_NO_DETAIL
+namespace detail
+{
+
+template <typename T>
+struct distance_measure
+{
+    T value;
+
+    distance_measure()
+        : value(T())
+    {}
+
+    bool is_small() const { return true; }
+    bool is_zero() const { return true; }
+    bool is_positive() const { return false; }
+};
+
+template <typename T>
+struct distance_measure_floating
+{
+    T measure;
+
+    distance_measure_floating()
+        : measure(T())
+    {}
+
+    // Returns true if the distance measure is small.
+    // This is an arbitrary boundary, to enable some behaviour
+    // (for example include or exclude turns), which are checked later
+    // with other conditions.
+    bool is_small() const { return std::fabs(measure) < 1.0e-3; }
```

Comment:
any clue about this number? does it come from experimentations?

Reply:
Also, probably: `math::abs(measure)`.

Reply:
The source is in the comments, right above - it is a more or less arbitrary number.
It is a FP number so fabs should be fine - but I could make it abs too

Reply:
Does this mean that changing this number affects the output/correctness or the performance of the algorithm?